### PR TITLE
Lorita initial upload

### PR DESCRIPTION
**Pull request by LChristelis**
The start of a Meerkat SDP readthedocs space. 
Other / existing user documentation is linked to this page.

### CODE REVIEW

**Comment by lmzxqtom** [comment]
Would be good to include a LICENCE.txt file. I'll add one onto the pull request.

**Comment by LChristelis** [comment]
@lmzxqtom I added the license file.

File: docs/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+import sphinx_rtd_theme
```
Comment (lmzxqtom) [inline docs/source/conf.py]: This is an unused import. Just checking why it's here.

**Review by lmzxqtom — APPROVED** [review]
Looks good! Just the unused import - not critical.

**Pull request merged**